src/cascadia/TerminalControl/TermControl.cpp

```diff
@@ -1159,12 +1159,24 @@ namespace winrt::Microsoft::Terminal::TerminalControl::implementation
     void TermControl::CopySelectionToClipboard(bool trimTrailingWhitespace)
     {
         // extract text from buffer
-        const auto copiedData = _terminal->RetrieveSelectedTextFromBuffer(trimTrailingWhitespace);
+        const auto bufferData = _terminal->RetrieveSelectedTextFromBuffer(trimTrailingWhitespace);
+
+        // convert text: vector<string> --> string
+        std::wstring textData;
+        for (const auto& text : bufferData.text)
+        {
+            textData += text;
+        }
+
+        // convert text to HTML format
+        const auto fontData = _terminal->GetFontInfo();

```

Comment:
I don't think you should be using `_terminal->GetFontInfo` here - I don't think the Terminal has any idea about its font. You should probably just use `_actualFont`